webpack.config.js

```diff
@@ -1,25 +1,23 @@
 const path = require('path');
 
 const commonJSConfig = {
-  entry: ['core-js/stable', 'regenerator-runtime/runtime', './src/index.js'],
```

Comment:
Why have you removed regenerator?

Reply:
Because we don't need it anymore. All tests are running in browsers as well.

Reply:
The test are running only on the latest stable browser versions, that doesn't mean that we don't need polyfills

Reply:
Decided that we will proceed without polyfils for now, in case of any issues in the future we'll bring it back.